README.md

```diff
@@ -1 +1,27 @@
-# NewPortfolio
\ No newline at end of file
+# MyPortfolio
+#This is MyPortfolio Project
+    This is simple Project.Which has header, Section.Inside header there is logo and menu icon.
+    Inside section some pictures. My itroduction in paragraph.
+
+  ![screenshot](./Images/newPort.png)
+  
+  ## Built With
+
+- Html,CSS
+- Git, Github
+- Visual Studio Code
+
+
+
+## Getting Started
+
+**This is an example of how you may create your mobile responsive porfolio**
+
+##Prerequisites
+
+Create repository
+Clone it local device using git
+Add files , commit and push them to github
+Run linters 
+Deploy to resolve errors
```

Comment:
- [ ] _[Optional]_ This part of your README should be either __`Usage`__ or __`Setup`__. It should also be a step-by-step instruction on how visitors can use or setup your project.